cardform/src/main/res/layout/issuer_item.xml

```diff
@@ -1,30 +1,42 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="@dimen/ui_dialog_title_height"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:background="?selectableItemBackground"
-    android:paddingTop="24dp"
     android:paddingBottom="24dp"
+    android:paddingTop="24dp"
```

Comment:
what do you think about creating a configuration in the dimens file for this 24dp?